### PR TITLE
fixes #446

### LINKED ISSUE #446: naming
https://github.com/math-comp/analysis/blob/f8eff34ded1ff4bd51883b56d7563e7cb408c181/theories/normedtype.v#L509

there are remaining uses of `minfty` in the names of lemmas

### PR DESCRIPTION
**Pull request by affeldt-aist**
TODO: issue to question the usefulness of `nbhs_pinfty_gt_pos`?

### CODE REVIEW

File: theories/normedtype.v [hunk 1]
```diff
@@ -498,25 +499,22 @@ Proof. by exists c%:num; split => // ; rewrite realE posnum_ge0. Qed.
...
-Proof. by exists c; split => //; apply: ltW. Qed.
+Lemma nbhs_pinfty_ge_real r : r \is Num.real -> \forall x \near +oo, r <= x.
+Proof. by exists r; split => //; apply: ltW. Qed.
 
-Lemma nbhs_minfty_lt (c : {posnum R}) : \forall x \near -oo, c%:num > x.
+Lemma nbhs_ninfty_lt (c : {posnum R}) : \forall x \near -oo, c%:num > x.
 Proof. by exists c%:num; split => // ; rewrite realE posnum_ge0. Qed.
 
-Lemma nbhs_minfty_le (c : {posnum R}) : \forall x \near -oo, c%:num >= x.
+Lemma nbhs_ninfty_le (c : {posnum R}) : \forall x \near -oo, c%:num >= x.
```
Comment (CohenCyril) [inline theories/normedtype.v:511]: While you're at it, I'd rephrase those with a precondition rather than an explicit `%:num`

File: theories/normedtype.v [hunk 2]
```diff
@@ -508,35 +509,29 @@ move=> [M [Mreal AM]]; exists (M * 2); split.
...
 
-Lemma nbhs_pinfty_ge (c : {posnum R}) : \forall x \near +oo, c%:num <= x.
-Proof. by exists c%:num; rewrite realE posnum_ge0; split => //; apply: ltW. Qed.
+Lemma nbhs_pinfty_ge_real r : r \is Num.real -> \forall x \near +oo, r <= x.
+Proof. by exists r; split => //; apply: ltW. Qed.
 
-Lemma nbhs_pinfty_gt_real (c : R) : c \is Num.real ->
-  \forall x \near +oo, c < x.
-Proof. by exists c. Qed.
+Lemma nbhs_pinfty_gt r : 0 < r -> \forall x \near +oo, r < x.
```
Comment (CohenCyril) [inline theories/normedtype.v:518]:
```suggestion
Lemma nbhs_pinfty_gt_pos r : 0 < r -> \forall x \near +oo, r < x.
```

Comment (CohenCyril) [reply theories/normedtype.v:518]: Is it necessary though? Isn't it subsumed by the `real` one? which could be the "main" one instead.

Comment (affeldt-aist) [reply theories/normedtype.v]: Both are actually used but it is true that there is such a small difference that this indicates a potential factorization. Should you put an issue about that?

**Review by CohenCyril — APPROVED** [review]
